### DISCUSSION TITLE
Share a component across different pages？

### DISCUSSION THREAD

**Discussion by pittleCheung** [GrapesJS/grapesjs#5152] [Q&A]
Hi experts,
I want to implement a function like webflow(https://webflow.com/), share the same component on different pages, when I change this common component, all pages that use this common component will also change。I hope to get some advice or ideas。Help

**Comment by artf** [comment]
Check this discussion https://github.com/GrapesJS/grapesjs/discussions/4317

**Comment by pittleCheung** [comment]
This plugin is really great, it really gave me some ideas
Below is my problem  

<img width="999" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/114127651/ac76d623-963a-4b21-8baa-c92eeab491d7">

I edited twice, each change will copy the currently changed component once and then add it to the editor, the first time I add 1 and the second time I add 2 on the basis of the first time, but it is added to the editor every time The value in the device is the last time

<img width="1396" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/114127651/a5f198f8-ab23-423a-8c5c-696a2ca24e4e">

<img width="1405" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/114127651/e9841707-ebf6-49f5-ad35-9052daddfae9">
